types/react-email-editor/index.d.ts

```diff
@@ -23,11 +24,30 @@ export interface User {
     readonly email?: string;
 }
 
-export interface MergeTag {
+export interface GroupedMergeTag {
+    readonly name: string;
+    readonly mergeTags: Array<SimpleMergeTag | GroupedMergeTag>;
+}
+
+export interface SimpleMergeTag {
     readonly name: string;
     readonly value: string;
 }
 
+export interface ConditionalMergeTagRule {
+    readonly name: string;
+    readonly before: string;
+    readonly after: string;
+}
+
+export interface ConditionalMergeTag {
+    readonly name: string;
+    readonly rules: ConditionalMergeTagRule[];
```

Comment:
I see on the [documentation page](https://docs.unlayer.com/docs/merge-tags#section-loops-and-conditions) that there is a "repeat" key inside the "rules" object, can it be used without it?

Reply:
it's the same thing as in `mergeTags` itself: they use object instead of array, assume the key is just a name of a rule or merge tag. So the form they refer in doc is:
```
mergeTags: {
   myTag: { value, name }
}
```
Since you originally used array for `mergeTags` types, I decided to do not break that and keep the same format. Since array in JS is almost an object -- it works that way